app/javascript/components/visualization/PlotTitle.jsx

```diff
@@ -1,6 +1,30 @@
 import React from 'react'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faInfoCircle } from '@fortawesome/free-solid-svg-icons'
+import { Popover, OverlayTrigger } from 'react-bootstrap'
+
+function makeGeneBadges(genes) {
+  return genes.map(gene => {
+    return <span className="badge popover-badge" key={gene}>{gene}</span>
+  })
+}
+
+// divide gene list into two parts, only showing first 3
```

Comment:
For hover tooltip in references, and less linter noise:

```suggestion
/** Divide gene list into two parts, only showing first 3 */
```